newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -78,7 +104,6 @@ export class ObjectEditorDialog extends Component<*, StateType> {
               resourceExternalEditors={this.props.resourceExternalEditors}
               onSizeUpdated={() =>
                 this.forceUpdate() /*Force update to ensure dialog is properly positionned*/}
-              objectName={this.props.objectName}
```

Comment:
Be careful, by removing this line you're breaking editors that relying on having `objectName` in their props.
Unfortunately the flow coverage is not good enough here to catch this error for you (you have to manually check by yourself the props taken by the editor to spot this error).
(In particular, if you remove this, the object name won't be available to name the sprites made with Piskel ;))

Reply:
ouch, sorry, I didn't realize . re-adding it 👍

Reply:
I think I must have removed it by mistake, sorry about that

Reply:
fixed now :) Good catch!